buddyboss/buddyboss-platform
# PROD-699 #3124 - Fix: Preview link is not working when no any keystroke or only using right-click to paste the URL

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3124 .

### How to test the changes in this Pull Request:

1. Copy any link from YouTube/Vimeo
2. Go to the News Feed and paste the URL using your mouse, do not add any text or use any keyboard shortcuts
3. click post and notice the preview is working fine

### Proof Screenshots or Video
https://vimeo.com/669320512/2b9829e112


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
